clamav/test_clamav_format.sh

```diff
@@ -12,6 +12,7 @@
 # limitations under the License.
 
 set -euo pipefail
+trap 'echo "Script errored at line ${LINENO}: ${BASH_COMMAND}"' ERR
```

Comment:
I don't think we need this, but I don't mind it.

I wanted to use trap call only for reporting error from tasks "which cannot fail"

Reply:
Deleted, thanks!